versions.lock

```diff
@@ -28,16 +30,16 @@ com.netflix.feign:feign-slf4j:8.18.0 (1 constraints: df080c87)
 com.palantir.conjure.java.api:errors:2.2.0 (1 constraints: a208407e)
 com.palantir.conjure.java.api:service-config:2.2.0 (1 constraints: a208407e)
 com.palantir.conjure.java.api:ssl-config:2.2.0 (2 constraints: 2f1b9d65)
-com.palantir.safe-logging:preconditions:1.8.1 (4 constraints: 7037b0e6)
-com.palantir.safe-logging:safe-logging:1.8.1 (6 constraints: 065913cf)
+com.palantir.safe-logging:preconditions:1.8.2 (4 constraints: 7137b1e6)
+com.palantir.safe-logging:safe-logging:1.8.2 (6 constraints: 085996cf)
 com.palantir.tokens:auth-tokens:3.3.0 (1 constraints: 9012f21d)
-com.palantir.tracing:tracing:2.6.0 (2 constraints: f81e13b6)
-com.palantir.tracing:tracing-api:2.6.0 (1 constraints: a00c4109)
-com.palantir.tracing:tracing-jersey:2.6.0 (1 constraints: a608507e)
-com.palantir.tracing:tracing-okhttp3:2.6.0 (1 constraints: a608507e)
+com.palantir.tracing:tracing:2.10.0 (2 constraints: 4e1f60ec)
+com.palantir.tracing:tracing-api:2.10.0 (3 constraints: 182c788e)
+com.palantir.tracing:tracing-jersey:2.10.0 (1 constraints: d108c286)
+com.palantir.tracing:tracing-okhttp3:2.10.0 (1 constraints: d108c286)
 com.palantir.tritium:tritium-registry:0.11.2 (1 constraints: 3605283b)
-com.squareup.okhttp3:logging-interceptor:3.13.1 (1 constraints: d608d786)
-com.squareup.okhttp3:okhttp:3.13.1 (5 constraints: 8745dcdf)
+com.squareup.okhttp3:logging-interceptor:3.14.0 (1 constraints: d608d786)
+com.squareup.okhttp3:okhttp:3.14.0 (5 constraints: b3456615)
```

Comment:
Decent number of changes went into https://github.com/square/okhttp/blob/master/CHANGELOG.md#version-3140. Seems like they're getting further and further away from a version of conscrypt that still works on Centos6 :/